webviz_examples/webviz-full-demo.yml

```diff
@@ -403,18 +403,15 @@ layout:
                   observation_file: *obsfile
                   ensembles: *hm_ensembles
 
-          # Some of the optional arguments are included as comments for illustration
-          # They must be activated/assigned if using non-default values,
-          # otherwise they can be removed for a cleaner appearance
           - page: Seismic misfit
             content:
               - SeismicMisfit:
                   ensembles: *hm_ensembles
                   attributes:
                     - topvolantis_amplitude_mean_20200701_20180101_1.txt
                     - basevolantis_amplitude_mean_20200701_20180101_1.txt
-                  # attribute_sim_path: sim2seis/output/4d_attribute_maps/  # relative to <runpath>
-                  # attribute_obs_path: ../../share/observations/seismic/  # relative to <runpath>
+                  attribute_sim_path: sim2seis/output/4d_attribute_maps/  # relative to <runpath>
+                  attribute_obs_path: ../../share/observations/seismic/  # relative to <runpath>
```

Comment:
Is this change intentional?

Reply:
Yes, it’s a preparational step to enable a change of the default path values in the plugin code without breaking the live demo.